vcbuild.bat

```diff
@@ -530,7 +530,7 @@ if not defined NODE_VERSION (
 if not defined DISTTYPE set DISTTYPE=release
 if "%DISTTYPE%"=="release" (
   set FULLVERSION=%NODE_VERSION%
-  exit /b 0
+  goto distexit
```

Comment:
Is there a reason the `goto` is needed? If it's only used in this one function can't you just do:

```diff
 if "%DISTTYPE%"=="release" (
     set FULLVERSION=%NODE_VERSION%
+   if not defined DISTTYPEDIR set DISTTYPEDIR=%DISTTYPE%
   exit /b 0
```

```diff
)
set FULLVERSION=%NODE_VERSION%-%TAG%
+if not defined DISTTYPEDIR set DISTTYPEDIR=%DISTTYPE%
```

It's possible I'm misunderstanding some of the subtleties of this script.